recirq/quantum_chess/quantum_board_test.py

```diff
@@ -78,7 +79,14 @@ def test_classical_jump_move(board):
     b.do_move(m)
     assert_samples_in(b, [u.squares_to_bitboard(['b1', 'c1'])])
 
-
+def test_path_qubits_error():
```

Comment:
Should we add a test for a path that is not an error?

Reply:
Thanks Doug it's added.